specs/results/koiosapi-guild.yaml

```diff
@@ -2806,7 +2803,7 @@ components:
                   description: Sum of minted assets (negative on burn)
                   example: 1
           metadata:
-            $ref: "#/components/schemas/tx_metadata/items/properties/metadata"
+            $ref: "#/components/schemas/tx_metadata"
```

Comment:
Looks good but I'm not sure I agree with this change, it ends up with the following format:
```
"metadata": [
      {
        "tx_hash": "string",
        "metadata": {
          "721": {
            "version": 1,
            "copyright": "...",
            "publisher": [
              "p...o"
            ],
            "4bf184e01e0f163296ab253edd60774e2d34367d0e7b6cbc689b567d": {}
          }
        }
      }
    ],
```
both the `tx_hash` and `metadata` fields in the object seem redundant (`tx_hash` is already available at top-level and metadata seems like a duplicated key)

Reply:
Thanks - good catch, not sure where this change came from, wasnt looking at modifying it indeed - fixed